keras_rs/src/utils/keras_utils.py

```diff
@@ -42,3 +42,16 @@ def check_shapes_compatible(
                 return False
 
     return True
+
+
+# Check ranks and shapes.
+def check_rank(
+    x_rank: int,
+    allowed_ranks: tuple[int, ...] = (1, 2),
```

Comment:
I wouldn't provide a default for `allowed_ranks`. When I'm reading the code, I don't want to have to go to the definition to know what ranks I'm checking, I should have them right there where it's called.